<launch>
    <include file=$(find turtlebot_bringup)/launch/minimal.launch/>
    <group ns="client">
        <node name="turn_robot_client" pkg="navigation" type="turn_robot_client"/>
    </group>
    <group ns="server">
        <node name="turn_robot_server" pkg="navigation" type="turn_robot_server"/>
    </group>
</launch>
